Better issue templates and stalebot (#1586)
MicrosoftDocs/edge-developer: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Report bugs with the Microsoft Edge documentation website

Reviewer: There's still a rather subtle distinction between "issue w/ the docs site" vs. "issue with the docs".  I had to take time to think about the difference; can spell out the contrast more explicitly like in a hallway conversation.
```suggestion
about: Report bugs with the Microsoft Edge documentation website, such as a broken link or Markdown tagging that's not working.
```

Reviewer: Move "report documentation content" to be above "documentation bug report".

Author: Agreed, we don't need the distinction between these 2 types of issues.

title: ''
labels: ''
assignees: ''
---

<!--

> ⚠ If your feedback is not about the Microsoft Edge developer documentation website, please consider submitting it here instead:

Reviewer: ```suggestion
⚠ If your feedback isn't about the Microsoft Edge developer documentation website, please consider submitting it here instead:
```
per Acrolinx & brand voice.
